common/djangoapps/course_action_state/migrations/0001_initial.py

```diff
@@ -4,7 +4,7 @@
 from django.db import migrations, models
 import django.db.models.deletion
 from django.conf import settings
-import xmodule_django.models
+from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
```

Comment:
Is it a problem to have common import from openedx? I guess it isn't an issue as long as the code doesn't have circularities if we consider that common is going away sooner than later.